server/apps/immich/src/api-v1/asset/asset-repository.ts

```diff
@@ -266,6 +267,11 @@ export class AssetRepository implements IAssetRepository {
       asset.tags = tags;
     }
 
+    if (asset.exifInfo != null) {
+      asset.exifInfo.description = dto.description || '';
+      await this.exifRepository.save(asset.exifInfo);
```

Comment:
Can you save exif info on the asset directly? asset repo.save(assetWithExif)

Reply:
No it cannot unfortunately